plugins/messages/lib/tasks/foodsoft.rake

```diff
@@ -19,11 +19,11 @@ namespace :foodsoft do
     hande_mail(ENV['RECIPIENT'], STDIN.read)
   end
 
-  desc "Start STMP server for incoming email (options: PORT=25, HOST=0.0.0.0)"
+  desc "Start STMP server for incoming email (options: PORT=2525, HOST=0.0.0.0)"
   task :reply_email_smtp_server => :environment do
-    port = ENV['PORT'].to_i
+    port = ENV['PORT'].present? ? ENV['PORT'].to_i : 2525
```

Comment:
since the environment variable is set, we start it now on port `3000` too

Reply:
Yess, that's a downside. 
Could use `SMTP_PORT` instead (in rake task), or put in the `Procfile` e.g.

    PORT=$SMTP_PORT bundle exec rake foodsoft:reply_email_smtp_server

What about `Dockerfile`'s `EXPOSE` here? Maybe ... it is cleaner to have a number of docker images, each work each type. I do find the overhead a bit too much, so right now I'm willing to make this trade-off.
What do you think?

Reply:
I'd prefere `SMTP_PORT` in the rake task. Docker EXPOSE ist not a real issue, since you can expose a port from the outside.

Reply:
Great, then I'll use `SMTP_PORT` in the rake task.

Reply:
`SMTP_PORT` would also be a good candidate for outgoing smtp settings (like [here](https://stackoverflow.com/a/45748832/2866660) and [here](https://www.openproject.org/operations/configuration/)), maybe `SMTP_SERVER_PORT`?

Reply:
agree!